frontend/src/libs/components/meditation-tracks-list/meditation-track/styles.module.scss

```
@@ -0,0 +1,81 @@
+.track {
+  position: relative;
+  width: 100%;
+  max-width: 451px;
+  height: 177px;
+  margin: 0 auto;
+  border-radius: 30px;
+  box-shadow: 0 4px 23px 0 var(--violet-100);
+}
+
+.background-image {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: block;
+  width: 37%;
+  height: 100%;
+  border-radius: 30px 0 0 30px;
+}
+
+.content {
+  position: absolute;
+  right: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 70%;
+  max-width: 85%;
+  height: 100%;
+  padding-right: 23px;
+  padding-left: 29px;
+  background-color: var(--white);
+  border-radius: 30px;
+}
+
+.info {
+  width: 52%;
+  height: 85px;
+  line-height: 20px;
+}
+
+.title {
+  margin: 0;
+  font-weight: 600;
+  font-size: 16px;
+}
+
+.duration {
+  display: inline-block;
+  margin-top: 8px;
+  padding: 6px 12px;
+  color: var(--blue-400);
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 16.8px;
+  text-align: center;
+  background-color: var(--blue-alpha-30);
+  border-radius: 14px;
+}
+
+.play-button {
+  display: block;
+  width: 48px;
+  height: 48px;
+  fill: var(--blue-300);
+}
+
+.play-button:hover {
+  transform: scale(1.1);
+  fill: var(--blue-400);
+}
+
+@media (width <= 600px) {
```

had  a similar mistake - it is better to use this version:
```suggestion
@media (width < 600px) {
```